numpy/core/numeric.py

```diff
@@ -2089,15 +2089,12 @@ def indices(dimensions, dtype=int):
     """
     dimensions = tuple(dimensions)
     N = len(dimensions)
-    if N == 0:
-        return array([], dtype=dtype)
+    shape = (1,)*N
     res = empty((N,)+dimensions, dtype=dtype)
     for i, dim in enumerate(dimensions):
```

Comment:
So since this meets line lengths rules, this should be good to merge? I'm guessing it's bad form to merge my own PR